geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy

```diff
@@ -0,0 +1,162 @@
+                            "CREATE INDEX IF NOT EXISTS id_blb ON $correlationTable($idFieldRsu); "+
+                            "DROP TABLE IF EXISTS $outputTableName; CREATE TABLE $outputTableName AS "+
+                            "SELECT SUM((1-a.$buContiguityColumn)*a.$buTotalFacadeLengthColumn*a.$height_wall)/"+
+                            "st_area(b.$geometricFieldRsu) AS rsu_free_external_facade_density, b.$idFieldRsu "
+
+            if(!inputColumns.isEmpty()){
+                query += ", a.${inputColumns.join(",a.")} "
+            }
+
+            query += "FROM $buildingTable a, $correlationTable b "+
+                        "WHERE a.$idFieldBu = b.$idFieldBu GROUP BY b.$idFieldRsu, b.$geometricFieldRsu;"
+
+            logger.info("Executing $query")
+            datasource.execute query
+            [outputTableName: outputTableName]
+        }
+)}
+
+/**
+ * Process used to compute the RSU ground sky view factor such as defined by Stewart et Oke (2012): ratio of the
+ * amount of sky hemisphere visible from ground level to that of an unobstructed hemisphere. The calculation is
+ * based on the ST_SVF function of H2GIS using the following parameters: ray length = 100, number of directions = 60
+ * and for a grid resolution of 10 m (standard deviation of the estimate calculated to 0.03 according to
+ * Bernard et al. (2018)). The density of points used for the calculation actually depends on building
+ * density (higher the building density, lower the density of points). To avoid this phenomenon, we set
+ * a constant density of point "point_density" for a given amount of free surfaces (default 0.008,
+ * based on the median of Bernard et al. (2018) dataset). The calculation needs the "rsu_building_density"
+ * and the "rsu_area".
+ *
+ * References:
+ * --> Stewart, Ian D., and Tim R. Oke. "Local climate zones for urban temperature studies." Bulletin of
+ * the American Meteorological Society 93, no. 12 (2012): 1879-1900.
+ * --> Jérémy Bernard, Erwan Bocher, Gwendall Petit, Sylvain Palominos. Sky View Factor Calculation in
+ * Urban Context: Computational Performance and Accuracy Analysis of Two Open and Free GIS Tools. Climate ,
+ * MDPI, 2018, Urban Overheating - Progress on Mitigation Science and Engineering Applications, 6 (3), pp.60.
+ *
+ * @return A database table name.
+ * @author Jérémy Bernard
+ */
+static IProcess rsuGroundSkyViewFactor() {
+    return processFactory.create(
+            "RSU ground sky view factor",
+            [rsuTable: String,inputColumns:String[],correlationBuildingTable: String,
+             rsuAreaColumn: String, rsuBuildingDensityColumn: String, pointDensity: Double, rayLength: Double,
+             numberOfDirection: Double, outputTableName: String, datasource: JdbcDataSource],
+            [outputTableName : String],
+            { rsuTable, inputColumns, correlationBuildingTable, rsuAreaColumn, rsuBuildingDensityColumn,
```

Comment:
I did not consider the inputColumns (potential columns that the user would like to keep) in the code.
Should we delete this inpuData ?

Reply:
Keep only an id field will be enough for me